dts/arm/atmel/samd21.dtsi

```diff
@@ -61,96 +61,54 @@
 			label = "WATCHDOG_0";
 		};
 
-		sercom0: uart@42000800 {
-			compatible = "atmel,sam0-uart";
+		sercom0: sercom@42000800 {
+			compatible = "atmel,sam0-sercom";
```

Comment:
I couldn't find this constrain in any yaml file.

Reply:
Fixed.